### PR TITLE
Completed es6-promises exercise.

### PR DESCRIPTION
**Pull request by leanne-vu**
Screen Shot 2022-12-09 at 10 17 07 AM

### CODE REVIEW

File: es6-promises/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const takeAChance = require('./take-a-chance');
+
+var hello = takeAChance('Leanne');
+
+hello.then(message => {
+  console.log(message);
+}).catch(message => {
+  console.log(message);
```
Comment (Zhiwutian) [inline es6-promises/index.js:8]: you want to log `message.message` here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

**Comment by leanne-vu** [comment]
Screen Shot 2022-12-09 at 10 34 21 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**